.github/workflows/tests.yml

```diff
@@ -87,12 +87,14 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: 3.8
-    - name: install flit
+    - name: install flit and twine
       run: |
-        pip install flit~=3.0
+        pip install flit~=3.0 twine
     - name: Build and publish
       run: |
-        flit publish
```

Comment:
I published `mdformat-tables` with flit 3.9.0, so there might be an issue with the generated token rather than that flit release:

https://github.com/executablebooks/mdformat-tables/blob/17e79139ed3c288ae8caba84ea834ee2c86c488c/.github/workflows/tests.yml#L90-L98

> https://github.com/executablebooks/mdformat-tables/actions/runs/10533763354/job/29190294084
> Successfully installed certifi-2024.7.4 charset-normalizer-3.3.2 docutils-0.20.1 flit-3.9.0 flit_core-3.9.0 idna-3.8 requests-2.31.0 tomli-w-1.0.0 urllib3-2.0.7

Would it be worthwhile to try generating a new PyPi token scoped to `mdformat-myst` before switching to twine?

Reply:
Ah ok, perhaps it was then just a matter of setting the proper auth ENV variables for flit publish. I tried to flit publish locally and entered credentials when asked for and got the error I linked to in flit's issue tracker. twine worked fine for me.

I think the token should work fine, but it was referred to by the wrong ENV variable name in tests.yml.

Reply:
Is there anything I can do to help with this?

Reply:
Thanks for offering, but I believe the current issue requires someone who is an admin of this repository to resolve

Reply:
I've pushed an alternative fix in #33 and will try to publish v0.2.1 to confirm

Reply:
Fixing the secret name resolved the issues with publishing (https://github.com/executablebooks/mdformat-myst/actions/runs/11828512545/job/32958768366). This PR could be closed unless the twine changes are preferred